airbyte-integrations/connectors/source-typeform/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY main.py ./
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.12
```

Comment:
this should be a minor version bump according to this [doc](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors)

Reply:
updated version with these changes